src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/Binders/ByteArrayModelBinder.cs

```diff
@@ -5,10 +5,10 @@
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Mvc.Internal;
 
-namespace Microsoft.AspNetCore.Mvc.ModelBinding
+namespace Microsoft.AspNetCore.Mvc.ModelBinding.Binders
 {
     /// <summary>
-    /// ModelBinder to bind Byte Arrays.
+    /// ModelBinder to bind byte Arrays.
     /// </summary>
     public class ByteArrayModelBinder : IModelBinder
```

Comment:
Weird case here. Previously would fall through to using an `ArrayModelBinder<byte>` which supports additional submitted formats. Probably need to instantiate this with that binder as a fallback -- and use it.